lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -148,6 +157,16 @@ class MongoClientInterface {
                 ignoreUndefined: true,
             });
             this.adminDb = client.db('admin');
+            // log cache hit/miss every 5min
+            this.cacheHitMissLoggerInterval = setInterval(() => {
```

Comment:
should we make these (cachehHit/miss) available as prometheus metrics?
this would be easier to consume and monitor...
Should probably be done later, in a separate PR.

Reply:
it's out of the scope of this PR, but yes good idea